README.md

```diff
@@ -45,11 +45,11 @@ A possible solution to this would be to create an extra APIs implementing the mi
 
 ## Requirements
 
-* PHP >= 5.4
+* PHP >= 7.3 or PHP 8.0
```

Comment:
```suggestion
* PHP >= 7.3
```

Reply:
I wanted to make clear that this library will be 8.0 ready. What about composer-like statement? `PHP ^7.3 || ^8.0`

Reply:
sure